docs/tutorial.md

```diff
@@ -187,20 +188,58 @@ func halfRoundedUp(n int) (res int) {
 ```
 
 Currently, the `pure` modifier has to be written after the pre and postcondition and before the `func` keyword. Furthermore, pure functions and methods have strict syntax constrains, the body must be a single return statement that returns a single deterministic side-effect free expression.
+Furthermore, `pure` functions must terminate. Thus, we add a `decreases` keyword instructing Gobra to prove termination as explained next.
+
+
+## Total Correctness
+
+By default, Gobra proves partial correctness, i.e., if a function terminates then its postcondition provably holds.
```

Comment:
```suggestion
By default, Gobra proves partial correctness, i.e., if a function terminates then its postcondition holds.
```
This word doesn't add much, and might induce confusion in people from different backgrounds.